proto/beacon/p2p/v1/messages.pb.go

```diff
@@ -5,10 +5,9 @@ package ethereum_beacon_p2p_v1
 
 import (
 	fmt "fmt"
+	proto "github.com/gogo/protobuf/proto"
```

Comment:
Are these changes from goimports ? i think we have them fixed

Reply:
I ran gazzlle --fix and update-go-pbs.sh
I didn't add this line

Reply:
Thats fine then